ui/app/scripts/routes.js

```diff
@@ -306,6 +306,24 @@ define(['./app'], function (dashboard) {
         feature: 'tasksEnabled'
       }
     })
+    .state('home.tasks.detailedTask', {
+      url : 'tasks/definitions/{taskName}',
+      templateUrl : taskTemplatesPath + '/detailed-task.html',
+      controller: 'DetailedTaskController',
+      data:{
+        authenticate: true,
+        feature: 'tasksEnabled'
+      }
+    })
+    .state('home.tasks.tabs.createComposedTask', {
+      url: '/createComposedJob',
```

Comment:
Nope. The endpoint used to be `tasks/composed-definitions` but it's gone now. The regular task definition creation endpoint `tasks/definitions` creates composed task definition now.